data/xml/2024.konvens.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <collection id="2024.konvens">
-  <volume id="main" ingest-date="2024-09-16" type="proceedings">
```

Comment:
Let's not change the ingestion date for an update. Can you edit this script to keep the old one? This is a common issue.